src/main/java/tutorlink/commandpackage/DeleteStudentCommand.java

```diff
@@ -1,7 +1,7 @@
 package tutorlink.commandpackage;
 
 import tutorlink.resultpackage.CommandResult;
-import tutorlink.studentpackage.StudentClass;
+import tutorlink.studentpackage.Student;
 
 public class DeleteStudentCommand extends FindStudentCommand {
```

Comment:
This should extend from from Command

Reply:
This command should also use the delete method from StudentList

Reply:
Hi I did not make changes to DeleteStudentCommand I believe this is flagged because I renamed StudentClass to Student.

Reply:
What i mean is that the delete command should work differently, as in stop inheriting from the FindStudentCommand and use the deleteStudent method of the studentList

Essentially, move the logic in this function to StudentList and have this command call it

Reply:
I agree, I didn't touch this file in my PR it only shows up due to me changing the name. Will settle this in the subsequent PR (or whoever is assigned to tackle this).